src/en/space-station-14/departments/medical/proposals/Virology Reworked.md

```diff
@@ -0,0 +1,72 @@
+
+## External Infection Sources
+Viruses that don't originate in the crew round start can be instead picked up from other sources, such as raw meats in the kitchen, not cleaning meat cutting tools in the kitchen, rats and mice, artifacts creating (usually very lethal) diseases, and sometimes external sources, such as carp bites, vent spider bites, spawned blood puddles, and potentially bio terror from nukkies.
+This is mostly done so the crew can't somehow systematically wipe out viruses.
+
+## Viral Modification
+Virologists can take multiple viruses and attempt to take characteristics from each of them in an effort to create new viruses, which can potentially be used to combat other viruses, or be turned into bioweappns.
+Viruses can be modified when exposed to Mutagen or Spacacillin to cause different effects.
+Mutagen will immediately trigger a mutation of the virus as would normally happen when the virus transfers between subjects. 
+Spacacillin can be used to "stabilize" the virus, and then later be used to intentionally infect people without the virus that the original sample was taken from. A Spacacillin modified virus would block the original virus from transferring to the patient as an alternative to a vaccine, as a form of inoculation instead but still subject the patient to the symptoms of the virus.
+
+## Contagion Tiers
+These can all stack in one virus based on volatility and mutation, and will always start with at least one of these.
+
+Blood to Blood: Requires direct blood contact to transfer, such as repeated syringe use, or the same melee weapon used against multiple people. Drinking blood may or may not count. Very low point cost.
+Saliva: Would require someone to drink or ingest the virus from an item someone else has drank or eaten off of without being cleaned first. Low point cost.
+Airborne, Inhalation: Would require people to stand near each other in order to transfer the virus. Can be blocked with breath masks. Moderate point cost.
+Contact: Virus can be picked up through contact with objects the infected has handled with a lifespan assigned to the virus for objects, as well as hugs, cuffing actions, buckling. Moderate point cost.
+Airborne, Infectious: High virulence, able to infect through anything via proximity except for full virology suits and the CMO hardsuit and Paramedic Softsuit (gotta make them useful for something). High point cost.
+All of the above can also have a species jump modifier, which will allow the virus to transfer between species and survive successfully, at a higher mutation cost.
+
+## Virus Symptoms
+Viruses should only very rarely be beneficial. They should be something that the crew should be aware of however. Viral symptoms would be generated based on the mutation point buy system with low cost mutations being most common. Ideally these symptoms would be assigned a severity as well.
```

Comment:
I was gonna leave this one to whoever coded it.

Reply:
I think also a larger variety of symptoms in general, but especially neutral- to-beneficial symptoms would be good to create RP scenarios that aren't just "hit violet alarm virus outbreak", and to incentivize virology gameplay other than systematically vaccinating every virus they find because all they can be is a liability. I agree that they should still be rare, but perhaps not extremely rare.